control-plane/agents/src/bin/core/volume/specs.rs

```diff
@@ -828,6 +829,31 @@ impl ResourceSpecsLocked {
         }
     }
 
+    pub(crate) fn check_capacity_limit_for_resize(
+        &self,
+        request: &ResizeVolume,
```

Comment:
we already have the limit from request, don't need to pass Option again

```suggestion
        cluster_limit: u64,
```